backend/lib/utils.js

```diff
@@ -0,0 +1,18 @@
+var nonKeyWords = ['the','and','with','has','had','his','hers','its','is','this','that','also',
+'just','who','where','while','when','why','well','have','been','not','out','from','but','for','you',
+'will','was','their','than','which','were','one','are','all','she','them','say','says','them','how',
+'because','other','see','can','no','us','may','could','theres','him','her','lot','hes',
+'shes','only','these']
+
+module.exports = {
+
+  removeUnwantedStrings: (keywordString) => {
```

Comment:
This is similar to what vanillasquad did in [cloud-reviews](https://github.com/vanillasquad/cloud-reviews/blob/master/src/guardian.js#L44-L53). Suggest something like this would be cleaner and [faster](http://jsperf.com/fac-string-parsing)
```javascript
(kwstr) => {
    return kwstr.replace(/[<>]/g, '') // strip tags
                .replace(/\s{2,}/g, ' ') // replace double whitespace
                .split(' ') // get array of words
                .filter(word => nonKeyWords.indexOf(word) === -1) // filter out nonKeyWords
                .join(' ') // form string again
  }
```